src/llmq/utils.h

```diff
@@ -57,8 +57,8 @@ namespace utils
 // includes members which failed DKG
 std::vector<CDeterministicMNCPtr> GetAllQuorumMembers(Consensus::LLMQType llmqType, const CBlockIndex* pQuorumBaseBlockIndex, bool reset_cache = false);
 
-void PreComputeQuorumMembers(const CBlockIndex* pQuorumBaseBlockIndex, bool reset_cache = false);
-uint256 GetHashModifier(const Consensus::LLMQParams& llmqParams, const CBlockIndex* pQuorumBaseBlockIndex);
+void PreComputeQuorumMembers(const CBlockIndex* pindex, bool reset_cache = false);
+uint256 GetHashModifier(const Consensus::LLMQParams& llmqParams, const CBlockIndex* pCycleQuorumBaseBlockIndex);
```

Comment:
My only "objection" is here. `GetHashModifier` is called for both non-rotated and rotated quorums, hence "cycle" is more adapted to rotation.

Reply:
for non-rotation quorums `pCycleQuorumBaseBlockIndex == pQuorumBaseBlockIndex` and since using `pQuorumBaseBlockIndex` for rotation ones is wrong using `pCycleQuorumBaseBlockIndex` for both is a good compromise imo